sdkit/generate/sampler/unipc_samplers.py

```diff
@@ -0,0 +1,107 @@
+import torch
+from torch import Tensor
+
+from .custom.unipc_sampler import UniPCSampler
+
+from sdkit import Context
+
+#unipc is highly customizable
+
+#variant: bh1, bh2, vary_coeff
+#time_skip: logSNR, time_uniform, time_quadratic
+#order: 1, 2, 3
+#lower_order_final: True, False
+#thresholding: True, False
+
+#TODO: find best suggestions for sample params
+samplers = {
+    'unipc': { #suggested version
+        'variant': 'bh1',
+        'time_skip': 'logSNR',
+        'order': 3,
+        'lower_order_final': True,
+        'thresholding': False,
+    },
+    'unipc_snr': { #same as suggested version
```

Comment:
Do we need duplicate entries? Maybe we can keep a single entry here (the snr one), and update the documentation to recommend one?